### DISCUSSION TITLE
Export Product error

### DISCUSSION THREAD

**Discussion by janiu-001** [dbeaver/dbeaver#14253] [DBeaver Development]
Hi export,

A issue reported when i plan to Use "Export  Product export Wizard", please check the detail info below, very appreciated to get you response. 

A cycle was detected when generating the classpath org.slf4j.api_1.7.30.v20200204-2150, ch.qos.logback.classic_1.2.3.v20200428-2012, org.slf4j.api_1.7.30.v20200204-2150.

if i check the  button "Allow for binary cycles in target platform" , blow error is reported.

A cycle was detected when generating the classpath org.jkiss.dbeaver.ext.greenplum_1.0.64.202108121540, org.jkiss.dbeaver.ext.postgresql_2.1.136.202108121540, org.jkiss.dbeaver.ext.greenplum_1.0.64.202108121540.

Thanks, 

Jason

**Comment by Matvey16** [comment]
@janiu-001 Hello, Jason. Please provide more information for us to reproduce this issue. Mainly your OS, DBeaver version, Database that you are using. And most importantly steps to reproduce with table DDL if it is possible.

**Comment by janiu-001** [comment]
OS: mac catalina
dbeaver: the latest master branch

i also have the same problem on the windows system.

Do you have a try to click ""Export Product export Wizard" in the view tab?It should be reproduced easily

**Comment by Matvey16** [comment]
@janiu-001 I still can't understand the exect steps. Do you mean you are exporting data from data view? If so, I need to know what is a target of your export. Another database or a file (csv, xml etc.)? It will be good to know what type of columns are you trying to export. And I can't understand what "Allow for binary cycles in target platform" means. If you are using Dbeave in non-english language, can you at least post a scrennshot with that setting highlighted, so I can understand what it is?

**Comment by janiu-001** [comment]
![1](https://user-images.githubusercontent.com/19970581/130421223-c021d1a8-dba8-4640-a957-4fdb404bc353.png)
![2](https://user-images.githubusercontent.com/19970581/130421250-b171aea0-5db7-4ba8-9073-ac7b86decdae.png)
![3](https://user-images.githubusercontent.com/19970581/130421279-81313945-bc1b-428e-8a70-9fdd22f63879.png)


Please check the attached image, the first two images are my procedure, the last one is the error

**Comment by tati-kru** [comment]
Why do you export product? You can build it with Maven and get binary artifacts from `target`.

**Comment by huyunduoduo** [comment]
oh
my
god
